guides/source/getting_started.md

```diff
@@ -1485,6 +1485,9 @@ Without this file, the confirmation dialog box wouldn't appear.
 
 ![Confirm Dialog](images/getting_started/confirm_dialog.png)
 
+TIP: Learn more about jQuery Unobtrusive Adapter (jQuery UJS) on
```

Comment:
@seuros Why? All the others references to another Rails guide are tips, not notes.